spec/support/page_objects/publish/courses/new_level_page.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module PageObjects
+  module Publish
+    module Courses
+      class NewLevel < PageObjects::Base
+        set_url "/publish/organisations/{provider_code}/{recruitment_cycle_year}/courses/level/new"
+
+        section :level_fields, '[data-qa="course__level"]' do
+          element :primary,           "#course_level_primary"
```

Comment:
weird tab spacing here

Reply:
fixed in https://github.com/DFE-Digital/teacher-training-api/pull/2471/commits/70f5a9167981cc9e75c455c54d1a694b6cd139d6